src/AccessibilityInsights.Rules/Rules.csproj

```diff
@@ -319,11 +316,6 @@
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>LocalizedControlTypeNames.Designer.cs</LastGenOutput>
     </EmbeddedResource>
-    <EmbeddedResource Include="Resources\Messages.resx">
```

Comment:
I would like to better understand the purpose of the removed Messages.resx file. Do the new documentation files make the old Messages.resx file obsolete?

Reply:
h

> > }
> 
> Line 52 should be indented one level
> 
> Refers to: src/AccessibilityInsights.Rules/CCAControlTypesFilter.cs:52 in [3cbffff](https://github.com/Microsoft/accessibility-insights-windows/commit/3cbffff0522ab342d7ba1e4473ade6a06e582469). [](commit_id = [3cbffff](https://github.com/Microsoft/accessibility-insights-windows/commit/3cbffff0522ab342d7ba1e4473ade6a06e582469), deletion_comment = False)

Somehow I can't see which file you're refering to.

Reply:
Pretty sure that Rob's response was meant for my comment in CCAControlTypesFilter.cs. Line 52 of that file is flush left and should be indented 4 spaces